crates/test_utils/tests/mod.rs

```diff
@@ -0,0 +1,6 @@
+#![allow(
```

Comment:
use tests.rs as file name, avoid using mod.rs conventions as we don't flow that.